x/wasm/keeper/stargate_bindings.go

```diff
@@ -0,0 +1,18 @@
+package keeper
+
+//DONTCOVER
+
+import (
+	"sync"
+)
+
+// StargateLayerBindings keeps whitelist and its deterministic
+// response binding for stargate queries.
+//
+// The query can be multi-thread, so we have to use
+// thread safe sync.Map instead map[string]bool.
+var StargateLayerBindings sync.Map
```

Comment:
> Also, is the a reason for not having this in query_plugins.go?

Nvm, I think it will start making sense once we start storing whitelists:
https://github.com/terra-money/classic-core/blob/53f66ce22a2e14abbf3ee575879cf8ca826998dc/x/wasm/types/query_whitelist.go#L14

> Suggest the name to contain "whitelist" in it.

Still think this makes sense

Reply:
Changed :)